cli_sdk.py

```diff
@@ -92,4 +97,21 @@ def get_tokens(intent, transcript):
             output.append(line)
         if separators == 2:
             break
-    print("\n".join(reversed(output)))
+    return reversed(output)
+
+
+def get_tokens(intent, transcript):
+    print("\n".join(get_token_logs(intent, transcript)))
+
+
+def get_token_coverage(intent, transcript):
+    token_lines = list(get_token_logs(intent, transcript))[3:-1]
+    tokens = [[w for w in t.split(" ") if w][-2:] for t in token_lines]
+    transcript = [w[-1] for w in tokens]
+    coverage = [("*" if w[0] != "None" else " ")*len(w[-1]) for w in tokens]
+    print()
+    print("Recognized parts of the transcript are denoted with '*'")
+    print()
+    print(' '.join(transcript))
+    print(' '.join(coverage))
+    print()
```

Comment:
Is this old python or could we make this a format print?
```python
    print("\nRecognized parts of the transcript are denoted with '*'\n\n{}\n{}\n\n.format(
        ' '.join(transcript)
        ' '.join(coverage)
        )
    )
```

Reply:
I'm not a huge fan of putting line breaks as `\n` in code since it makes the code less readable.

I updated the PR with a commit to make it a single print statement. There's some extra tabs, but that should prevent YAPF from screwing up in the future.

Reply:
I think your new version looks better than mine too.